tools/ci_build/build.py

```diff
@@ -649,8 +657,13 @@ def run_onnxruntime_tests(args, source_dir, ctest_path, build_dir, configs, enab
             run_subprocess([sys.executable, 'onnxruntime_test_python.py'], cwd=cwd, dll_path=dll_path)
 
             if args.enable_training and args.use_cuda:
+                # run basic frontend tests
```

Comment:
"enable_frontend_training_tests" sounds like it controls whether training frontend tests (even basic ones) are enabled. would something like "enable_all_training_frontend_tests" be more appropriate?